Orange-OpenSource/Orange-Boosted-Bootstrap
# Prepare v5.0.0-beta2

- Bootstrap v5.0.0-beta2 release: https://github.com/twbs/bootstrap/releases/tag/v5.0.0-beta2
- Bootstrap blog post: https://blog.getbootstrap.com/2021/02/10/bootstrap-5-beta-2/

---

Preview : https://deploy-preview-625--boosted.netlify.app

---

  - Firefox ESR
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - ~~if bumping a minor or major version~~ → still v5.0
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - Ship list: #585
- [ ] merge on `v5-dev`
  - attach zip file
  - paste CHANGELOG in the release's description
- [ ] run `build/ship.sh x.x.x` to…
  - Pull the latest changes update new version numbers
  - Compile CSS and JS
  - Compress dist files
  - Generate the hashes and update our config's cdn section
  - Compile the docs
  - Copy-pasta built docs to `bs-docs`-named clone of Boosted.
  - [ ] copy `../bs-docs` to the `gh-pages` branch
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
